README.md

```diff
@@ -1,8 +1,17 @@
-# Celestia Networks Repository
+# Celestia Networks
 
-This repository contains the `genesis.json` for Celestia testnets.
+This repository contains the configuration files for Celestia networks.
 
-If you want to see versions per testnets that are being used, then navigate to this [grafana dashboard](https://celestia.grafana.net/public-dashboards/5d14d96e44f04664bb0c44267e5d645c)
+Please refer to the [Celestia Docs](https://docs.celestia.org) for guides on running your own node. The configuration files `genesis.json` and `addrbook.json` are intended for use by Consensus nodes and should be placed in the node's home directory (by default `~/.celestia-app/config`).
```

Comment:
Pedantic, but `~` (tilde expansion) is a shell-specific syntax, and isn't necessarily portable between shells (though in practice it is), and doesn't apply to non-shell environments.

```suggestion
Please refer to the [Celestia Docs](https://docs.celestia.org) for guides on running your own node. The configuration files `genesis.json` and `addrbook.json` are intended for use by Consensus nodes and should be placed in the node's home directory (by default `$HOME/.celestia-app/config`).
```